README.md

```diff
@@ -46,11 +46,12 @@ We recommend using [`buf-setup-action`][buf-setup] to install it (as in the exam
 ## Configuration
 
 | Parameter           | Description                                                                    | Required | Default                             |
-| :------------------ | :----------------------------------------------------------------------------- | :------- | :---------------------------------- |
+| :------------------ |:-------------------------------------------------------------------------------| :------- | :---------------------------------- |
 | `buf_token`         | The [Buf authentication token][buf-token] used for private [Buf inputs][input] | ✅       | [`${{github.token}}`][github-token] |
 | `input`             | The path of the [input] you want to push to BSR as a module                    |          | `.`                                 |
 | `draft`             | Indicates if the workflows should push to the BSR as a [draft][buf-draft]      |          |                                     |
 | `create_visibility` | The visibility to create the BSR repository with, if it does not already exist |          |                                     |
+| `tag`               | The custom tag to push to the BSR                                              |          |                                     |
```

Comment:
Currently, we don't support "moving"/"re-writing" the same tag to a different commit. So if a user tries to set a single, static tag, their subsequent pushes will fail. This should be called out here.

Suggestion on this documentation:

" A custom tag to push with each commit. The BSR does not currently support moving or duplicated tags, so use this with care."